modules/carbonAnalyticsAdapter.js

```diff
@@ -0,0 +1,227 @@
+import { generateUUID } from '../src/utils.js';
+import { ajax } from '../src/ajax.js';
+import { getStorageManager } from '../src/storageManager.js';
+import {getGlobal} from '../src/prebidGlobal.js';
+import adapterManager from '../src/adapterManager.js';
+import adapter from '../libraries/analyticsAdapter/AnalyticsAdapter.js';
+import CONSTANTS from '../src/constants.json';
+
+const CARBON_GVL_ID = 493;
+const ANALYTICS_VERSION = 'v1.0';
+const CARBON_ANALYTICS_URL = 'https://pb-ing.ccgateway.net';
+const PROFILE_ID_KEY = 'carbon_ccuid';
+const PROFILE_ID_COOKIE = 'ccuid';
+const SESSION_ID_COOKIE = 'ccsid';
+const MODULE_NAME = 'carbon';
+const ANALYTICS_TYPE = 'endpoint';
+
+const DAY_MS = 24 * 60 * 60 * 1000;
+const MINUTE_MS = 60 * 1000;
+
+export const storage = getStorageManager({gvlid: CARBON_GVL_ID, moduleName: 'carbon'});
+
+let enabledEvents = [];
```

Comment:
so we will require the pub to set something in their config for this? No default?